packages/design-system/theme/src/_tokens.scss

```diff
@@ -355,6 +366,15 @@
 		var(--color-sticky-default-border-l)
 	);
 
+  	--color-white: hsl(0, 0, 100, 1);
+
+	@each $color in ('primary', 'neutral', 'secondary', 'success', 'warning', 'danger') {
```

Comment:
would be nice if we could demo these as well in demo pages

Reply:
These are still up for discussion. Max said we'll add numbered shades in a later iteration. Should I comment them out?

Reply:
ya if we are not using them anywhere and Max is not sold, let's comment them out or delete them.